generators/PHPythia8/PHPythia8.C

```diff
@@ -254,12 +284,26 @@ int PHPythia8::process_event(PHCompositeNode *topNode)
 
   // save statistics
   if (_integral_node)
-  {
-    _integral_node->set_N_Generator_Accepted_Event(_pythia->info.nAccepted());
-    _integral_node->set_N_Processed_Event(_eventcount);
-    _integral_node->set_Sum_Of_Weight(_pythia->info.weightSum());
-    _integral_node->set_Integrated_Lumi(_pythia->info.nAccepted() / (_pythia->info.sigmaGen() * 1e9));
-  }
+    {
+      if(_legacy_vertexgenerator)
+            
+	{
+	  _integral_node->set_N_Generator_Accepted_Event(ppGen->info.nAccepted()+pnGen->info.nAccepted());
+	  _integral_node->set_N_Processed_Event(_eventcount);
+	  _integral_node->set_Sum_Of_Weight(ppGen->info.weightSum()+pnGen->info.weightSum());
+	  _integral_node->set_Integrated_Lumi((ppGen->info.nAccepted()+pnGen->info.nAccepted()) / ((ppGen->info.sigmaGen()+pnGen->info.sigmaGen()) * 1e9));
```

Comment:
The right formula should be `ppGen->info.nAccepted() / ppGen->info.sigmaGen() + pnGen->info.nAccepted() / pnGen->info.sigmaGen()`.